quantum/keymap_extras/keymap_italian.h

```diff
@@ -166,7 +166,7 @@
 
 // DEPRECATED
 #define IT_BKSL IT_BSLS
-#define IT_QUOT IT_APOS
+//#define IT_QUOT IT_APOS
```

Comment:
```suggestion
#define IT_APOS IT_QUOT
```

It should actually be reversed, as these are for backward compatibility. My bad...